game-core/src/main/java/games/strategy/engine/xml/TestAttachment.java

```diff
@@ -47,15 +46,15 @@ public String getName() {
   public void setName(final String name) {}
 
   public void setValue(final String value) {
-    m_value = value;
+    this.value = value;
   }
 
   public String getValue() {
-    return m_value;
+    return value;
   }
 
   public void resetValue() {
-    m_value = null;
+    value = null;
```

Comment:
We really need to figure out a way on how to move this to test code.
Maybe by mocking the mapping map?